docs/standard/threading/how-to-register-callbacks-for-cancellation-requests.md

```diff
@@ -1,28 +1,33 @@
 ---
-title: "How to: Register Callbacks for Cancellation Requests"
-ms.date: "03/30/2017"
+title: Register callbacks for cancellation requests
+ms.date: 08/14/2020
+ms.topic: how-to
```

Comment:
@IEvangelist I confirmed that it's set correctly from @tdykstra PR in the [current live](https://docs.microsoft.com/en-us/dotnet/standard/threading/how-to-register-callbacks-for-cancellation-requests).

The live page HTML contains:

```html
<meta name="ms.topic" content="how-to" />
```

Reply:
@Youssef1313 For each page, telemetry tracks metrics such as "bounce rate", "dwell time (time spent on the page)", "click-through rate", etc. Different doc types have different expectations of what's expected. The doc type determines a formula that calculates the "performance" of each individual page when compared to other pages of the same type.